src/lib/fragments/NglMoleculeViewer.react.js

```diff
@@ -22,12 +23,19 @@ export default class NglMoleculeViewer extends Component {
     }
 
     componentDidMount() {
-        const {id, stageParameters, viewportStyle} = this.props;
+        const {id, stageParameters, viewportWidth, viewportHeight} = this.props;
         const params = {...stageParameters};
         const stage = new Stage(id, params);
         const orientationMatrix = stage.viewerControls.getOrientation();
 
-        stage.setSize(viewportStyle.width, viewportStyle.height);
+        const widthStr = isNaN(viewportWidth)
```

Comment:
`isNaN` has some weird properties - for example it thinks whitespace strings are numbers (because `+' ' === 0`. So please just use `fast-isnumeric`.

Reply:
I thought `isNaN` might be a little cleaner so we're importing fewer modules, but `isNumeric` does make more sense and removes ambiguity in this case. Updated in c39b8a6.